legendary/cli.py

```diff
@@ -40,6 +40,8 @@
 
 class LegendaryCLI:
     def __init__(self, override_config=None, api_timeout=None):
+        if not stdout.isatty():
+            stdout.reconfigure(encoding='utf-8')
```

Comment:
You probably need to set this to the locale variable? I think windows does not support utf-8 for example.

Reply:
This is not used when output is displayed to terminal, but when it is piped to file or another application. In this case, it's reasonable to expect that next application is supporting UTF8.

The reason this PR is created is that Python already opens stdout with default encoding from locale, and it cannot parse some Unicode symbols being output.

Reply:
@derrod plz take a look